### DISCUSSION TITLE
Do you have to use yarn?

### DISCUSSION THREAD

**Discussion by elliott-w** [keystonejs/keystone#7708] [Questions]
Can you use npm to create a new project?

I tried running `npx create-keystone-app` and `npm init keystone-app`, but of those commands still used yarn to install dependencies

**Comment by woutertje99** [comment]
In my case, NPM worked fine. It goes through a couple steps, using Yarn first. If this is not found, it skips Yarn and goes to NPM. If you have Yarn installed, it will use that for the initialization. Can you uninstall it, init the project, then install it back? That would probably work!

**Reply by elliott-w** [reply]
Yeah after install I just deleted the `yarn.lock` file and ran `npm install` and everything seems to work fine.
